Sources/Bot/API/GitHubAPIProtocol.swift

```diff
@@ -1,21 +1,22 @@
 import ReactiveSwift
 import Result
 
+// TODO: Refactor interface
```

Comment:
I tried to minimise the changes here but some where helpful to avoid unnecessary mappings.